app/poros/movie.rb

```diff
@@ -1,3 +1,42 @@
 class Movie
+  attr_reader :id,
+              :title,
+              :vote_average,
+              :runtime,
+              :genres,
+              :summary,
+              :cast,
+              :total_reviews,
+              :reviews,
+              :image
+  def initialize(data)
```

Comment:
I'm using rating @rating = attributes['vote_average'] on mine so I will probably need that added back in.

Reply:
Okay. We can change @vote_average to @rating instead, I'll have to update my tests accordingly as well